package.json

```diff
@@ -2,14 +2,13 @@
   "name": "meriyah",
   "version": "5.0.0",
   "description": "A 100% compliant, self-hosted javascript parser with high focus on both performance and stability",
-  "main": "dist/meriyah.cjs",
-  "module": "dist/meriyah.esm.js",
-  "jsnext:main": "dist/meriyah.esm.js",
+  "main": "dist/meriyah.cjs.js",
+  "module": "dist/meriyah.esm.mjs",
   "browser": "dist/meriyah.umd.min.js",
   "unpkg": "dist/meriyah.umd.min.js",
   "exports": {
     "import": "./dist/meriyah.esm.mjs",
-    "require": "./dist/meriyah.cjs",
+    "require": "./dist/meriyah.cjs.js",
     "types": "./dist/src/meriyah.d.ts"
```

Comment:
I know current setting works , but TS  docs says

> The "types" condition should always come first in "exports".

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-7.html#packagejson-exports-imports-and-self-referencing

Let's move it to top.